saleor/extensions/__init__.py

```diff
@@ -1,10 +1,12 @@
 import importlib
-from typing import List
+from typing import Dict, List, NewType, Union
 
 from django.utils.translation import pgettext_lazy
 
 from .checks import check_extensions  # NOQA: F401
 
+ConfigurationType = NewType("ConfigurationType", Dict[str, Union[str, bool]])
```

Comment:
Please correct me If I am wrong. You use it only in the tests, am I right?  Shouldn't be declared inside tests in that case?